backend/database/models/products.js

```diff
@@ -3,13 +3,16 @@ const products = (sequelize, DataTypes) => {
     name: DataTypes.STRING,
     description: DataTypes.STRING,
     image: DataTypes.STRING,
-    votes: DataTypes.ARRAY,
```

Comment:
there should be no votes field in products table, since there is a table for votes. I hope.

Reply:
so, this looks logical.

Reply:
actually, that's true let me work on it